.github/workflows/gradle-check.yml

```diff
@@ -43,7 +63,7 @@ jobs:
           files: ./codeCoverage.xml
 
       - name: Create Comment Success
-        if: success()
+        if: ${{ github.event_name == 'pull_request_target' && success() }}
```

Comment:
Isn't this also needed for `push` event ?

Reply:
no